seed-server/network-connection.sh

```diff
@@ -82,16 +102,49 @@ sed -e "s/\${SERVICE_NETWORK_ADDRESS}/${service_network_address}/" \
     -e "s/\${POD_NETWORK_NETMASK}/${pod_network_netmask}/" \
     /client-config-dir/vpn-shoot-client.template > /client-config-dir/vpn-shoot-client
 
-if [[ ! -z "$node_network" ]]; then
-    for n in $(echo $node_network |  sed 's/[][]//g' | sed 's/,/ /g')
+if [[ -n $is_ha ]]; then
+    for ((i=0; i < $HA_VPN_CLIENTS; i++))
```

Comment:
Do we want to perform a check earlier whether the environment variable is set properly? I guess it is set via the deployment, but the script will behave strange if it is not set, e.g. running it locally. It would be hard to set a proper default, though. Therefore, you could probably only check for existence and fail early with a proper error message.

What do you think.

Reply:
The variable `is_ha` is only set if the env variable `POD_NAME` ends with a digit (`-[0-4]$`).
So if you run it locally, this branch will not be executed.
Anyway, I did not check for running it locally. If you think it's important, we should have a shared session to make the script robust for that case.